src/lib/common/string.cpp

```diff
@@ -676,3 +676,41 @@ void strReplace(char* to, int toLen, const char* from, const char* oldString, co
 
   to[toIx] = 0;
 }
+
+
+
+/* ****************************************************************************
+*
+* servicePathCheck - 
+*/
+std::string servicePathCheck(const char* servicePath)
```

Comment:
We have another function (in rest.cpp) with the same name:

    int servicePathCheck(ConnectionInfo* ciP, const char* servicePath)

The functions don't do the same, however it is probably true I could use the 'new' function inside the old function.